app/bundles/ApiBundle/Security/Permissions/ApiPermissions.php

```diff
@@ -47,15 +48,19 @@ public function getName()
      */
     public function buildForm(FormBuilderInterface &$builder, array $options, array $data)
     {
-        $builder->add('api:access', 'permissionlist', [
-            'choices' => [
-                'full' => 'mautic.api.permissions.granted',
-            ],
-            'label'  => 'mautic.api.permissions.apiaccess',
-            'data'   => (!empty($data['access']) ? $data['access'] : []),
-            'bundle' => 'api',
-            'level'  => 'access',
-        ]);
+        $builder->add(
+            'api:access',
+            PermissionListType::class,
+            [
+                'choices' => [
+                    'full' => 'mautic.api.permissions.granted',
```

Comment:
This should be flipped

Reply:
Please use `choices_as_values` attr

Reply:
Flipped in all places and the choices_as_values added to the class itself. Thanks for noticing!